client/components/admin/includes/header-navbar.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom'
+import { addCategory } from '../../../actions/book_actions';
+
+
+export default class HeaderSideBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      description: '',
+    }
+    this.onChange = this.onChange.bind(this);
+    this.handleFormSubmit = this.handleFormSubmit.bind(this);
+  }
+
+  handleFormSubmit(e) {
+    e.preventDefault();
+    addCategory(this.state)
+      .then((message) => {
+        Materialize.toast(message, 2000, 'blue');
+        window.location.href = '/admin';
+      })
+      .catch((err) => err)
+  }
+  onChange(e) {
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+  render() {
+    return (<div className="admin-header-side" id="container">
+      <ul id='dropdown1' className='dropdown-content'>
+        <li><a href="#!"> {this.props.username}</a></li>
+        <li className="divider"></li>
+        <li><a onClick={this.props.onClick} href="#"><i className="material-icons">exit_to_app</i> Logout</a></li>
+      </ul>
+      <div id="menu">
+        <div style={{ float: 'right', color: '#fff', padding: 5 }}>
+          <a style={{ float: 'right', marginLeft: 5, backgroundColor: '#2962ff' }}
+            className='dropdown-button btn' href='#' data-activates='dropdown1'>Account</a>
+        </div>
+        <div className="col s3">
+          <ul id="slide-out" className="side-nav fixed show-on-large-only">
+            <div style={{ textAlign: 'center', color: '#fff', backgroundColor: 'blue', marginTop: -16 }}>
```

Comment:
You may want to take these styles into a variable(object) and then access them in the elements

Reply:
Thanks, I will surely fix it up.